Removed unsupported versions from database products (#5764)
EnterpriseDB/docs: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx

> For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:

Reviewer: ```suggestion
For more information about using the `pg_ctl` utility, or the command line options available, see the official [PostgreSQL Core Documentation](https://www.postgresql.org/docs/12/static/app-pg-ctl.html).
```

> [https://www.postgresql.org/docs/12/static/app-pg-ctl.html]<https://www.postgresql.org/docs/12/static/app-pg-ctl.html>

Reviewer: ```suggestion
```

> For more information about using the command line client, see the PostgreSQL Core Documentation at:

Reviewer: ```suggestion
For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/12/static/app-psql.html).
```

[https://www.postgresql.org/docs/12/static/app-psql.html]<https://www.postgresql.org/docs/12/static/app-psql.html>

## Creating a Database Cluster and Starting the Service

The PostgreSQL `initdb` command creates a database cluster; when installing Advanced Server with an RPM package, the `initdb` executable is in `/usr/edb/asx.x/bin`. After installing Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:

-   Specify environment options on the command line.
-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.

> To review the `initdb` documentation, visit:

Reviewer: ```suggestion
For more information, see the [initdb documentation](https://www.postgresql.org/docs/12/static/app-initdb.html).
```

> [https://www.postgresql.org/docs/12/static/app-initdb.html]<https://www.postgresql.org/docs/12/static/app-initdb.html>

Reviewer: ```suggestion
```

For more information about using `initdb`, and the available cluster configuration options, see the PostgreSQL Core Documentation available at:

> [https://www.postgresql.org/docs/12/static/app-initdb.html]<https://www.postgresql.org/docs/12/static/app-initdb.html>

Reviewer: ```suggestion
```
